addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -310,6 +310,7 @@ odoo.define('website.s_website_form', function (require) {
                             break;
                     }
 
+                    self.restoreBtnLoading();
```

Comment:
- In case of a redirection to another page, this should not be called
- In case of scroll to an anchor, I would restore only after the scroll, otherwise we see the form button flicker at the same time the page scroll, feels unperfect to me

Reply:
@qsm-odoo I made the change